mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/wfs.servers.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Yes, the ```js``` dir should be reserved for code, not config.

I know GEOR_custom.js is stored in the js folder ... it should probably not be.